tooling-tests/src/test/resources/test-src/output/tool_test_generate_14/Dependencies.toml

```diff
@@ -0,0 +1,86 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
+
```

Comment:
Same

Reply:
updated in [eac1fc8](https://github.com/ballerina-platform/persist-tools/pull/23/commits/eac1fc89ac6d1d3cd9a9387fce669b33074b9ced)